services/provider/server/consumer.go

```diff
@@ -79,6 +79,8 @@ func (c *ocsConsumerManager) Create(ctx context.Context, onboard ifaces.StorageC
 		Status: ocsv1alpha1.StorageConsumerStatus{
 			Client: ocsv1alpha1.ClientStatus{
 				OperatorVersion: onboard.GetClientOperatorVersion(),
+				ClusterID:       onboard.GetClusterId(),
+				NamespacedName:  onboard.GetNamespacedName(),
```

Comment:
1. there is an existing bug here, updating the status here doesn't stick because the storageconsumer controller is probably resetting it, yet to be investigated, however operatorversion is refreshed by hearbeat mechanism and so this was a bit safer for operatorversion but not for the additional params
2. if by mistake the status is cleaned additional params will get erased w/ no way of retrieving them and so I feel status section for these constant fields isn't a good idea

Reply:
Whether it'd stick or not in the long run shouldn't be of any concern here since those fields' values are queried at the very moment a `StorageConsumer` resource gets created (it's watched by `StorageCluster`). From that point forward it's set in the CM.

Reply:
@leelavg I am not sure about #1, but you do have a point with #2. 
From the start, I pushed toward NOT sending the data again on every heartbeat (and only doing it once as part of the onboarding), but that didn't takes into consideration that status should not be a source of truth. 

With that in mind, maybe we should move all of these fields to the heartbeat instead of the onboarding message. @raaizik what do you think?